bin/db-converter/index.js

```diff
@@ -0,0 +1,60 @@
+import './program.js';
+import { ObjectId } from 'mongodb';
+import { closeConnection, getFromLocalDB, saveData } from './lib.js';
+
+console.log('Start converting...');
+const [pages, aliases, files, pagesOrder] = ['pages', 'aliases', 'files', 'pagesOrder'].map(getFromLocalDB);
+
+const pagesIdsMap = pages.reduce((acc, curr) => {
+  const newId = new ObjectId();
+
+  acc.set(curr._id, newId);
+
+  return acc;
+}, new Map());
+
+// Explicitly set the root page id
+pagesIdsMap.set('0', '0');
```

Comment:
is it ok that '0' is not an `ObjectId`?

Reply:
I don't see any problems for now with this solution. It works OK